app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -137,7 +155,7 @@ const mapStateToProps = ({
     column: BACKLOG,
     stories,
     project,
-    pastIterations
+    pastIterations    
```

Comment:
Can you remove these empty spaces?

```suggestion
    pastIterations
```